src/components/commentInput/commentInput.tsx

```diff
@@ -87,13 +88,15 @@ export const CommentInput: FC<CommentInputProps> = ({
                 onChange={(ev: React.ChangeEvent<HTMLTextAreaElement>): void =>
                   setUserComment(ev.target.value)}
               />
-              <button type="button" className="BigCard__button" onClick={submitCommentHandler}>
-                Add comment
-              </button>
+              <Button
+                className=".btn"
```

Comment:
Šeit nevajag to "BigCard__button"

Reply:
Manuprāt, es izņēmu ārā vispār BigCard_button, jo man liekas, ka lielās kartiņas poga īpaši neatšķirās. Tagad visās vietās ir komponente ar savu scss stilu.